#request
sensor_msgs/Image image

#pixels
int64 imgDimX 

#pixels
int64 imgDimY

float64 mapPixelsPerMeter

#number of horizontal edges (squares+1)
int64 chessboardEdgesH

#number of vertical edges (squares+1)
int64 chessboardEdgesV

#meters
float64 squareWidth

---

#response: indication of whether the chessboard was identified
bool x